Dockerfile

```diff
@@ -1,9 +1,10 @@
 FROM tensorflow/tensorflow:2.7.1
 
-COPY requirements.txt /app/requirements.txt
-RUN pip install -r /app/requirements.txt
+COPY requirements.txt ./requirements.txt
+RUN pip install --no-cache-dir -r ./requirements.txt
 
-COPY . /app/
-WORKDIR /app/
+COPY selection/ ./selection
+COPY dataperf_speech_config.yaml ./dataperf_speech_config.yaml
+COPY allowed_training_set.yaml ./allowed_training_set.yaml
```

Comment:
Good point!

Easy fix for the .yaml files but I'm not sure how to copy in selection.py while mounting  /selection on run.

Any advice?

Reply:
the idea in my mind is to copy the whole directory in (hence why we should explicitly discourage downloading the embeddings inside the repo dir, otherwise the image will be 30gb) so selection.py will already be present in the built container... does that not work?